<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>RecorderDocker</class>
 <widget class="QWidget" name="RecorderDocker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>338</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>277</width>
        <height>338</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>14</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetSettings" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="layoutSnapshots" stretch="0,0">
            <item>
             <widget class="QLabel" name="labelDirectory">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string comment="A folder where recordings are stored">Recordings directory:</string>
              </property>
              <property name="buddy">
               <cstring>buttonBrowse</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonManageRecordings">
              <property name="toolTip">
               <string>Manage recordings...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="folder">
                <normaloff>.</normaloff>
                .
               </iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutDir">
            <item>
             <widget class="QLineEdit" name="editDirectory">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonBrowse">
              <property name="toolTip">
               <string>Select the recordings directory...</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="folder">
                <normaloff>.</normaloff>
                .
               </iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item column="0" row="0">
             <widget class="QLabel" name="labelCaptureRate">
              <property name="text">
               <string>Capture interval:</string>
              </property>
              <property name="buddy">
               <cstring>spinCaptureInterval</cstring>
              </property>
             </widget>
            </item>
            <item column="1" row="0">
             <widget class="QSpinBox" name="spinCaptureInterval">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string comment="Contraction of &quot;second&quot; for interval"> sec.</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item column="0" row="1">
             <widget class="QLabel" name="labelQuality">
              <property name="text">
               <string comment="Compression quality">Quality:</string>
              </property>
              <property name="buddy">
               <cstring>spinQuality</cstring>
              </property>
             </widget>
            </item>
            <item column="1" row="1">
             <widget class="KisSliderSpinBox" name="spinQuality" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item column="0" row="2">
             <widget class="QLabel" name="labelResolution">
              <property name="text">
               <string comment="Resolution for recorded frames">Resolution:</string>
              </property>
              <property name="buddy">
               <cstring>comboResolution</cstring>
              </property>
             </widget>
            </item>
            <item column="1" row="2">
             <widget class="QComboBox" name="comboResolution">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Original</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Half</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Quarter</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxRecordIsolateMode">
            <property name="toolTip">
             <string>Record frames when Isolate Layer or Isolate Group mode is active.
Depending on timelapse speed this may cause flickering in resulting video.</string>
            </property>
            <property name="text">
             <string comment="Record when Isolate Layer mode is active">Record in isolate mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAutoRecord">
            <property name="toolTip">
             <string extracomment="Tooltip for check box">Start recording after creating a new image or opening an existing one.</string>
            </property>
            <property name="text">
             <string>Record automatically</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutButtons">
         <item>
          <widget class="QPushButton" name="buttonRecordToggle">
           <property name="text">
            <string>Record</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonExport">
           <property name="text">
            <string>Export...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelUnsupportedColorSpace">
         <property name="text">
          <string>⚠ Unsupported color space.
&quot;RGB/Alpha 8-bit&quot; is only supported.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisSliderSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>